blueoil/cmd/init.py

```diff
@@ -139,22 +139,22 @@
 def network_name_choices(task_type):
     if task_type == 'classification':
         return [definition['name'] for definition in classification_network_definitions]
-    elif task_type == 'object_detection':
+    if task_type == 'object_detection':
```

Comment:
I'm not sure this change is what @tsawada san suggested.
`elif` looks no problem in my opinion...

Reply:
```python
defs = {
  'classification': classification_network_definitions,
  'object_detection': object_detection_network_definitions,
  ...
}
if task_type not in defs:
  raise ValueError
return [... for ... in defs[task_type]
```

Reply:
@iizukak The `el(se)` here is semantically redundant and I prefer to use `if`, but it's not necessary, so I'll undo it if you don't like it.

@tsawada That's a good idea. Is it appropriate to apply it in this PR?

Reply:
I think it's better to use tables here.

Reply:
fixed